python/cudf/cudf/core/column/column.py

```diff
@@ -2329,3 +2344,10 @@ def concat_columns(objs: "MutableSequence[ColumnBase]") -> ColumnBase:
 
     # Filter out inputs that have 0 length, then concatenate.
     return libcudf.concat.concat_columns([o for o in objs if len(o)])
+
+
+def _has_any_nan(arbitrary):
```

Comment:
I'd prefer to define this function above where it is used, rather than below.

Reply:
Done 👍